redbot/core/cli.py

```diff
@@ -25,7 +41,7 @@ def confirm(text: str, default: Optional[bool] = None) -> bool:
             value = input(f"{text}: [{options}] ").lower().strip()
         except (KeyboardInterrupt, EOFError):
             print("\nAborted!")
-            sys.exit(1)
+            sys.exit(ExitCodes.INVALID_CLI_USAGE)
```

Comment:
This doesn't match with other places where this error code is used. This is generally used for invalid usage of CLI flags, not the user ctrl c-ing. Other places which handle ctrl c, like when the user is prompted to enter their token, it uses `SHUTDOWN`. I think this makes more sense, since the user is explicitly trying to stop whatever is happening, even if it's done forcefully.

Reply:
This is also a handler for `EOFError` which can occur in a non-interactive context where STDIN is closed. Therefore, it's possible that when used in a script, `redbot-setup` wasn't supposed to be aborted but due to incorrect usage it was. We could perhaps split this into two exception handlers instead though.